src/modules/roc_packet/units.h

```diff
@@ -87,6 +87,27 @@ static inline size_t num_channels(channel_mask_t ch_mask) {
     return n_ch;
 }
 
+//! FEC block number in a packet stream.
+typedef uint16_t blknum_t;
+
+//! FEC block numbers difference.
+typedef int16_t blknum_diff_t;
+
+//! Compute difference between two FEC block numbers.
+inline blknum_diff_t blknum_diff(blknum_t a, blknum_t b) {
```

Comment:
Please add tests for blknum helpers to roc_packet/test_units.cpp

Reply:
I was very creative and just stole your tests for seqnum_t .